Cargo.lock

```diff
@@ -4,11 +4,11 @@ version = 3
 
 [[package]]
 name = "addr2line"
-version = "0.16.0"
+version = "0.17.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3e61f2b7f93d2c7d2b08263acaa4a363b3e276806c68af6134c44f523bf1aacd"
+checksum = "b9ecd88a8c8378ca913a680cd98f0f13ac67383d35993f86c90a70e3f137816b"
 dependencies = [
- "gimli",
+ "gimli 0.26.1",
```

Comment:
I wonder how this happened? Now the lockfile includes 2 versions of gimli (dwarf parser). I guess this is from upgrade to the backtrace crate. 

I suggest not doing that upgrade here in the spirit of minimizing the diff.